install/create_cabinets_yaml.md

```diff
@@ -20,14 +20,18 @@ cabinets:
     - id: 1001
       nmn-vlan: 2001
       hmn-vlan: 3001
-    - id: 1002
-    - id: 1003
+    - id: 1100
+      nmn-vlan: 2100
+      hmn-vlan: 3100
+    - id: 1101
+      nmn-vlan: 2101
+      hmn-vlan: 3101
 - type: river
   total_number: 4
   starting_id: 3000
 ```
 
-In this example file, there are two Hill cabinets that will be automatically numbered as 9000 and 9001. The Mountain cabinets appear in three groupings of four IDs. The River cabinets are non-contiguous in four separated IDs.
+In this example file, there are two Hill cabinets that will be automatically numbered as 9000 and 9001. The Mountain cabinets appear in two groups as 1000 and 1001 and then another area in 1100 and 1101 with VLAN ids related to the cabinet number. This is one way to provide space in the cabinet IDs to allow for future expansion of the system or to identify location of a row of cabinets.  The River cabinets are contiguous in four separated IDs as 3000, 3001, 3002, and 30003..
```

Comment:
Change second sentence to:

The Mountain cabinets appear in two groups as 1000 and 1001, and then in another area as 1100 and 1101, with VLAN IDs related to the cabinet number.